iroh-unixfs/src/builder.rs

```diff
@@ -411,6 +452,8 @@ impl Default for DirectoryBuilder {
             name: None,
             entries: Default::default(),
             typ: DirectoryType::Basic,
+            chunker: None,
```

Comment:
should be the same as the default for `File`

Reply:
ahh, yea, it should also be able to take a `degree` as well, just like `FileBuilder`

thanks